r_examples/r_api_serving_examples/API Serving Examples.ipynb

```diff
@@ -0,0 +1,881 @@
+{
```

Comment:
<p>Instead of committing the data in the repository, please use the AWS-hosted version of the original dataset at <code>s3://sagemaker-sample-files/datasets/tabular/iris/iris.data</code>. That version is .data instead of .csv, and does not have headers. If you need a copy identical to the one you committed, I can upload it.</p><p> </p><p>Please cite the UCI ML repository and link to their Iris entry: https://archive.ics.uci.edu/ml/datasets/iris</p><p>Dua, D. and Graff, C. (2019). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml]. Irvine, CA: University of California, School of Information and Computer Science.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>